Source/WebCore/platform/mediastream/cocoa/AudioMediaStreamTrackRendererUnit.h

```diff
@@ -66,6 +63,7 @@ class AudioMediaStreamTrackRendererUnit : public BaseAudioMediaStreamTrackRender
 
 private:
     AudioMediaStreamTrackRendererUnit();
+    friend std::unique_ptr<AudioMediaStreamTrackRendererUnit> std::make_unique<AudioMediaStreamTrackRendererUnit>();
```

Comment:
could make a static method here to return the unique_ptr rather than having to use `friend` would remove the style error too.